internal/client.go

```diff
@@ -195,7 +195,7 @@ func (c *Client) Options(path string) (classes map[string]bool, methods map[stri
 func (c *Client) SyncCollection(path, syncToken string, level Depth, limit *Limit, prop *Prop) (*Multistatus, error) {
 	q := SyncCollectionQuery{
 		SyncToken: syncToken,
-		SyncLevel: string(level),
+		SyncLevel: string(rune(level)),
```

Comment:
In this case we don't want to convert to an ASCII character, we want to format the level as a number. `level.String()` is designed for this.

Reply:
Oh, that's fixed now